test/CadlRanchProjects/type/union/src/Generated/EnumsOnly.cs

```diff
@@ -140,7 +141,7 @@ public virtual async Task<Response> SendAsync(EnumsOnlyCases prop, CancellationT
         {
             Argument.AssertNotNull(prop, nameof(prop));
 
-            SendRequest6 sendRequest6 = new SendRequest6(prop);
+            SendRequest6 sendRequest6 = new SendRequest6(prop, null);
```

Comment:
Do we really need to add an explicit `null ` here?

Reply:
this is for the raw data field.
Raw data field parameter will appear in every serialization ctor if MRW is enabled.
Previously we do not have to provide this because we are using initialization ctor.

Reply:
Since this part is totally generated by the generator, and the serialization ctor is always internal, we do not need to worry about it.